etc/docker/tests/run_test.sh

```diff
@@ -3,7 +3,8 @@ mkdir build
 cd build
 cmake .. -G Ninja -DVIAMCPPSDK_USE_DYNAMIC_PROTOS=ON \
 		-DVIAMCPPSDK_OFFLINE_PROTO_GENERATION=ON \
-		-DVIAMCPPSDK_SANITIZED_BUILD=ON
+		-DVIAMCPPSDK_SANITIZED_BUILD=ON \
+		-DVIAMCPPSDK_CLANG_TIDY=ON
```

Comment:
I have added the llvm apt repo to the docker files that support it and have pinned us to clang-tidy-13. This will help keep all users on a similar clang-tidy version and help prevent surprises with version conflicts in CI.